docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>You'll need to update from master I think.</p><p> </p><p>Can you add Acemoglu, Johnson and Robinson (2001) to <code>references.bib</code> and add as a proper citation? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done. Both integrate with master and updated .bib